gpp/src/utils/schemas/document.js

```diff
@@ -327,6 +326,28 @@ export default {
     },
     "year_type": {
       "type": "string"
+    },
+    "language": {
+      "type": "string",
+      "enum": [
+        "english",
```

Comment:
We have to make a note somewhere to update this if we ever need to change the languages available